### DISCUSSION TITLE
Is it possible to create a "Glossary" page, based on Graph Acryonms?

### DISCUSSION THREAD

**Discussion by HiltonGiesenow** [microsoft-search/pnp-modern-search#3642] [Q&A]
I know we can query the Acronym database in M365 Search using PnP Modern Search, but it doesn't seem possible to query for ALL results (e.g. "*"). Of course I want to focus this just on result type (i.e. Acronyms), as this would enable me to build a dynamic "glossary" of all company acronyms. Does someone know a way to query Graph in this way?

**Comment by wobba** [comment]
https://learn.microsoft.com/en-us/graph/api/search-searchentity-list-acronyms?view=graph-rest-1.0&tabs=http except it's an admin API really and requires a Read Write scope.

**Reply by HiltonGiesenow** [reply]
Thanks Michael. There's nothing in PnP Search that leverages this though, right? So I'd be totally outside of PnP, like in a custom SFPx web part?

**Reply by wobba** [reply]
Nothing in this project to dump all acronyms, only trigger and show what is returned when searching. And yes, a custom web part would be needed.

**Reply by HiltonGiesenow** [reply]
thanks @wobba , that's what I figured in the end - no worries, will roll my own of something
